_examples/sitemap/sitemap.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
Perhaps this example could have a bit specific name, e.g. `shopify_sitemap`. `sitemap` is too general for me. What do you think?